draft-ietf-core-oscore-edhoc.md

```diff
@@ -210,7 +210,11 @@ EDHOC verification                                           |
 
 To this end, the specific approach defined in this section consists of sending a single EDHOC + OSCORE request, which conveys the pair (C_R, EDHOC message_3) within an OSCORE-protected CoAP message.
 
-That is, the EDHOC + OSCORE request is in practice the OSCORE Request from {{fig-non-combined}}, as still sent to a protected resource and with the correct CoAP method and options intended for accessing that resource. At the same time, the EDHOC + OSCORE request also transports the pair (C_R, EDHOC message_3) required for completing the EDHOC session. Note that, as specified in {{client-processing}}, C_R is transported in the OSCORE Option rather than in the request payload.
+That is, the EDHOC + OSCORE request is composed of following two parts combined together in a single CoAP message.
+
+* The OSCORE Request from {{fig-non-combined}}, which is also in this case sent to a protected resource, with the correct CoAP method and options intended for accessing that resource.
+
+* EDHOC data consisting of the pair (C_R, EDHOC message_3) required for completing the EDHOC session. Note that, as specified in {{client-processing}}, C_R is transported in the OSCORE Option of the OSCORE Request rather than in the request payload.
```

Comment:
Should you mention that the EDHOC data are "prepended" (not just "in") the request payload?

Reply:
Of the EDHOC data, only EDHOC message_3 is prepended to the payload of the OSCORE Request, while C_R is already specified in the OSCORE Option of that OSCORE Request.

The former point was related to the second from last paragraph of Section 3.0, now moved within the bullet point on the EDHOC data that also clarifies the prepending.